Fotogjengen/hilfling-frontend
# Fixed Remote Router Dom Update

Changes:
- fix app to work with react rout dom v6
- fix error in PhotGangBangerApi

How to test:
- Check if app is running without error on every route
- Check that there is no error in /about when running with backend

Routes to check:
```
        <Route path="/" element={<App />} />
        <Route path="/motive" element={<ShowMotive />} />
        <Route path="/motive/:id" element={<ShowMotive />} />
        <Route path="/about" element={<About />} />
        <Route path="/search" element={<Search />} />
        <Route path="/search/:term" element={<Search />} />
        <Route path="/myprofile" element={<MyProfile />} />
        <Route path="/csa-tester" element={<CsaTester />} />
        <Route path="/intern/last-opp" element={<PhotoUpload />} />
        <Route path="/intern/arkivsjef" element={<Arkivsjef />} />
        <Route element={<NotFound />} />
```
